performance-tests/src/libs/backbone-client/identity.ts

```diff
@@ -36,30 +36,31 @@ export function createIdentity(client: Httpx, clientId: string, clientSecret: st
     }
 }
 
-export function exchangeToken(client: Httpx, createdIdentityResponse: CreateIdentityResponse, password: string) : TokenResponse {
+export function exchangeToken(client: Httpx, username: string, password: string): JwtResponse {
     const payload = {
         /* eslint-disable @typescript-eslint/naming-convention */
         client_id: "test",
         client_secret: "test",
         grant_type: "password",
-        username: createdIdentityResponse.device.username,
+        username,
         password
         /* eslint-enable @typescript-eslint/naming-convention */
     };
-    return client.post("http://localhost:8081/connect/token", payload, {
-        headers: {
-            // eslint-disable-next-line @typescript-eslint/naming-convention
-            "Content-Type": "application/x-www-form-urlencoded"
-        }
-    }).json() as unknown as TokenResponse;
+    return client
+        .post("http://localhost:8081/connect/token", payload, {
```

Comment:
I don't like the hardcoded domain here. This makes it impossible to configure it. Instead, we should do it similarly to how we do it in the SDK:
* The base url should be just the domain (without a path)
* There should be a constant containing the api version
* The api version should be appended on every request (if applicable)